<<<<<<< HEAD
2016/04/10 16:36:50 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`siadmdb`.`riwayat_penyakit`, CONSTRAINT `fk_riwayatpenyakit_santri` FOREIGN KEY (`nip_santri`) REFERENCES `santri` (`nip`) ON DELETE CASCADE ON UPDATE CASCADE). The SQL statement executed was: INSERT INTO `riwayat_penyakit` (`nip_santri`, `nama_penyakit`, `tanggal`) VALUES (:yp0, :yp1, :yp2).
in C:\xampp\htdocs\siadm\protected\controllers\RiwayatPenyakitController.php (73)
in C:\xampp\htdocs\siadm\index.php (13)
2016/04/10 16:36:50 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`siadmdb`.`riwayat_penyakit`, CONSTRAINT `fk_riwayatpenyakit_santri` FOREIGN KEY (`nip_santri`) REFERENCES `santri` (`nip`) ON DELETE CASCADE ON UPDATE CASCADE). The SQL statement executed was: INSERT INTO `riwayat_penyakit` (`nip_santri`, `nama_penyakit`, `tanggal`) VALUES (:yp0, :yp1, :yp2)' in C:\xampp\htdocs\yii\db\CDbCommand.php:358
Stack trace:
#0 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(1080): CDbCommand->execute()
#1 C:\xampp\htdocs\yii\db\ar\CActiveRecord.php(809): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\siadm\protected\controllers\RiwayatPenyakitController.php(73): CActiveRecord->save()
#3 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): RiwayatPenyakitController->actionCreate()
#4 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('riwayatPenyakit...')
#14 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\siadm\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/siadm/index.php?r=riwayatPenyakit/create
HTTP_REFERER=http://localhost/siadm/index.php?r=riwayatPenyakit/create
---
2016/04/10 14:01:17 [error] [exception.CException] exception 'CException' with message 'Property "LoginForm.Nama Pengguna" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\web\helpers\CHtml.php(2729): CComponent->__get('Nama Pengguna')
#1 C:\xampp\htdocs\yii\web\helpers\CHtml.php(2452): CHtml::resolveValue(Object(LoginForm), 'Nama Pengguna')
#2 C:\xampp\htdocs\yii\web\helpers\CHtml.php(1534): CHtml::activeInputField('text', Object(LoginForm), 'Nama Pengguna', Array)
#3 C:\xampp\htdocs\yii\web\widgets\CActiveForm.php(792): CHtml::activeTextField(Object(LoginForm), 'Nama Pengguna', Array)
#4 C:\xampp\htdocs\siadm\protected\views\site\login.php(29): CActiveForm->textField(Object(LoginForm), 'Nama Pengguna')
#5 C:\xampp\htdocs\yii\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\yii\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\web\CController.php(782): CController->renderPartial('login', Array, true)
#9 C:\xampp\htdocs\siadm\protected\controllers\SiteController.php(98): CController->render('login', Array)
#10 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#11 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('login')
#15 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#16 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\siadm\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/siadm/index.php?r=site/login
---
2016/04/11 01:39:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "login".' in C:\xampp\htdocs\yii\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('login')
#1 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\siadm\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/siadm/index.php/login
---
2016/04/11 01:39:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "UserLogin".' in C:\xampp\htdocs\yii\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('UserLogin')
#1 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\siadm\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/siadm/index.php/UserLogin
---
2016/04/11 01:40:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "login".' in C:\xampp\htdocs\yii\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('login')
#1 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\siadm\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/siadm/index.php/login
---
2016/04/11 01:40:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "loginform".' in C:\xampp\htdocs\yii\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('loginform')
#1 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\siadm\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/siadm/index.php/loginform
---
2016/04/11 01:48:15 [warning] [application] Failed to set unsafe attribute "username" of "LoginForm".
in C:\xampp\htdocs\siadm\protected\controllers\SiteController.php (93)
in C:\xampp\htdocs\siadm\index.php (13)
2016/04/11 01:48:15 [error] [exception.CException] exception 'CException' with message 'Property "LoginForm.namapengguna" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\validators\CRequiredValidator.php(58): CComponent->__get('namapengguna')
#1 C:\xampp\htdocs\yii\validators\CValidator.php(201): CRequiredValidator->validateAttribute(Object(LoginForm), 'namapengguna')
#2 C:\xampp\htdocs\yii\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#3 C:\xampp\htdocs\siadm\protected\controllers\SiteController.php(95): CModel->validate()
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#5 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('login')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#10 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\siadm\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/siadm/index.php/site/login
HTTP_REFERER=http://localhost/siadm/index.php/site/login
---
2016/04/11 01:48:19 [warning] [application] Failed to set unsafe attribute "username" of "LoginForm".
in C:\xampp\htdocs\siadm\protected\controllers\SiteController.php (93)
in C:\xampp\htdocs\siadm\index.php (13)
2016/04/11 01:48:19 [error] [exception.CException] exception 'CException' with message 'Property "LoginForm.namapengguna" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\validators\CRequiredValidator.php(58): CComponent->__get('namapengguna')
#1 C:\xampp\htdocs\yii\validators\CValidator.php(201): CRequiredValidator->validateAttribute(Object(LoginForm), 'namapengguna')
#2 C:\xampp\htdocs\yii\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#3 C:\xampp\htdocs\siadm\protected\controllers\SiteController.php(95): CModel->validate()
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#5 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('login')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#10 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\siadm\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/siadm/index.php/site/login
HTTP_REFERER=http://localhost/siadm/index.php/site/login
---
2016/04/11 01:48:25 [warning] [application] Failed to set unsafe attribute "username" of "LoginForm".
in C:\xampp\htdocs\siadm\protected\controllers\SiteController.php (93)
in C:\xampp\htdocs\siadm\index.php (13)
2016/04/11 01:48:25 [error] [exception.CException] exception 'CException' with message 'Property "LoginForm.namapengguna" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\validators\CRequiredValidator.php(58): CComponent->__get('namapengguna')
#1 C:\xampp\htdocs\yii\validators\CValidator.php(201): CRequiredValidator->validateAttribute(Object(LoginForm), 'namapengguna')
#2 C:\xampp\htdocs\yii\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#3 C:\xampp\htdocs\siadm\protected\controllers\SiteController.php(95): CModel->validate()
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#5 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('login')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#10 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\siadm\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/siadm/index.php/site/login
HTTP_REFERER=http://localhost/siadm/index.php/site/login
---
2016/04/11 01:49:23 [warning] [application] Failed to set unsafe attribute "username" of "LoginForm".
in C:\xampp\htdocs\siadm\protected\controllers\SiteController.php (93)
in C:\xampp\htdocs\siadm\index.php (13)
2016/04/11 01:49:23 [error] [exception.CException] exception 'CException' with message 'Property "LoginForm.namapengguna" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\validators\CRequiredValidator.php(58): CComponent->__get('namapengguna')
#1 C:\xampp\htdocs\yii\validators\CValidator.php(201): CRequiredValidator->validateAttribute(Object(LoginForm), 'namapengguna')
#2 C:\xampp\htdocs\yii\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#3 C:\xampp\htdocs\siadm\protected\controllers\SiteController.php(95): CModel->validate()
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#5 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('login')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#10 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\siadm\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/siadm/index.php/site/login
HTTP_REFERER=http://localhost/siadm/index.php/site/login
---
2016/04/11 01:49:31 [warning] [application] Failed to set unsafe attribute "username" of "LoginForm".
in C:\xampp\htdocs\siadm\protected\controllers\SiteController.php (93)
in C:\xampp\htdocs\siadm\index.php (13)
2016/04/11 01:49:31 [error] [exception.CException] exception 'CException' with message 'Property "LoginForm.namapengguna" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\validators\CRequiredValidator.php(58): CComponent->__get('namapengguna')
#1 C:\xampp\htdocs\yii\validators\CValidator.php(201): CRequiredValidator->validateAttribute(Object(LoginForm), 'namapengguna')
#2 C:\xampp\htdocs\yii\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#3 C:\xampp\htdocs\siadm\protected\controllers\SiteController.php(95): CModel->validate()
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#5 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('login')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#10 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\siadm\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/siadm/index.php/site/login
HTTP_REFERER=http://localhost/siadm/index.php/site/login
---
2016/04/11 03:30:43 [warning] [application] Failed to set unsafe attribute "username" of "LoginForm".
in C:\xampp\htdocs\siadm\protected\controllers\SiteController.php (93)
in C:\xampp\htdocs\siadm\index.php (13)
2016/04/11 03:30:43 [error] [exception.CException] exception 'CException' with message 'Property "LoginForm.namapengguna" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\validators\CRequiredValidator.php(58): CComponent->__get('namapengguna')
#1 C:\xampp\htdocs\yii\validators\CValidator.php(201): CRequiredValidator->validateAttribute(Object(LoginForm), 'namapengguna')
#2 C:\xampp\htdocs\yii\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#3 C:\xampp\htdocs\siadm\protected\controllers\SiteController.php(95): CModel->validate()
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#5 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('login')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#10 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\siadm\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/siadm/index.php/site/login
HTTP_REFERER=http://localhost/siadm/index.php/site/login
---
2016/04/11 03:31:05 [warning] [application] Failed to set unsafe attribute "username" of "LoginForm".
in C:\xampp\htdocs\siadm\protected\controllers\SiteController.php (93)
in C:\xampp\htdocs\siadm\index.php (13)
2016/04/11 03:31:05 [error] [exception.CException] exception 'CException' with message 'Property "LoginForm.namapengguna" is not defined.' in C:\xampp\htdocs\yii\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\validators\CRequiredValidator.php(58): CComponent->__get('namapengguna')
#1 C:\xampp\htdocs\yii\validators\CValidator.php(201): CRequiredValidator->validateAttribute(Object(LoginForm), 'namapengguna')
#2 C:\xampp\htdocs\yii\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#3 C:\xampp\htdocs\siadm\protected\controllers\SiteController.php(95): CModel->validate()
#4 C:\xampp\htdocs\yii\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#5 C:\xampp\htdocs\yii\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\web\CWebApplication.php(282): CController->run('login')
#9 C:\xampp\htdocs\yii\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#10 C:\xampp\htdocs\yii\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\siadm\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/siadm/index.php/site/login
HTTP_REFERER=http://localhost/siadm/index.php/site/login
---
=======
>>>>>>> master
